iroh-embed/README.md

```diff
@@ -0,0 +1,21 @@
+# iroh embed
+
+This provides an API to embed
```

Comment:
can you add the various shields that we have in the other readmes please?

Reply:
sure, i have a personal dislike for shields though ;)